src/walk-path.js

```diff
@@ -24,7 +24,6 @@ class WalkPath extends React.Component {
         />
         <Polyline
           className={className}
-          interactive=false
```

Comment:
Ah, I noticed the error and changed it to `interactive="false"`, which results in `interactive == true`; hooray for weak typing. Changing it to `interactive={false}` does indeed work :). I'll re-revert.